.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,15 +1,22 @@
 ---
 name: Bug Report
 about: Report a bug/crash
-title: ''
+title: '[Bug] A brief description about problem you found'
 labels: ''
 assignees: ''
 
 ---
 
+<!--
 **IMPORTANT:** Before continuing, please ensure that Building Gadgets (and any other mods involved) is updated to the latest available version
 ----------------------------------------------------------------------------
 After deleting this template, please provide the following information:
 * Forge Version
-* *If updating was not possible*; Building Gadgets version
+* Building Gadgets version 
+  * **NEVER** put "lastest", new releases might come out as you post the issue
+  * If updating crashes the game, clearly state that in your description
```

Comment:
```suggestion

```
Since this is the only bullet point left, it can be incorporated into the "If the issue is a crash..." line. I've done that below.